### ISSUE TITLE
Comparison with AnchorFX

### ISSUE THREAD

**Issue by antoine-guillou** [RobertBColton/DockFX#25]
Hi Robert,
A discussion on the comparison with AnchorFX: https://github.com/alexbodogit/AnchorFX/issues/20#issuecomment-263724971
It would be interesting to have your point of view.
Cheers

**Comment by RobertBColton** [comment]
What I was doing when I created this project was really to just create a reference implementation to get something out there so that it wouldn't be years for JavaFX to get a docking framework. I do a lot of hobby programming and I'm always building interfaces, so a docking framework is something that I expected to be extremely useful to myself, and I wanted to make sure it was available. I think it's pretty clear I succeeded in doing so because I helped spawn several of these other docking frameworks, which is good. Very happy to see that as the result.

I see that AnchorFX does claim to support tabs (I haven't tested it myself), something I haven't gotten to yet. So if these are features you are looking for then it's a great alternative to DockFX, which is really meant to keep it simple and make sure it continues to work in future Java versions. Also the licensing is compatible as well, so really I guess it just comes down to whichever works for your particular project.

I hope that answers your questions!
